### PR TITLE
ENH: Added MRI-PROSTATEx, MRA-Head, and MRI-Cardiac data

### PR DESCRIPTION
**Pull request by aylward**
Added data to data.kitware.com.

Added thumbnails and descriptions.

Provided proper citation in About dialog.

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -123,32 +125,50 @@ export const LABELMAP_PALETTE = {
...
       '2.16.840.1.114362.1.11972228.22789312658.561585527.143.7.60.651251220200409.1D000000S0D000000S0D000000S0D000000S1D000000S0D000000',
   },
+  {
+    name: 'MRA Head and Neck',
+    filename: 'MRA-Head_and_Neck.zip',
+    description: 'MRA from Patient Contributed Image Repository. Click application help icon "(?)" for more info. (15 MB)',
+    url: 'https://data.kitware.com/api/v1/item/6352a2b311dab8142820a33b/download',
+    image: MRAHeadThumbnail,
+    volumeKey:
+      '1.2.276.0.50.192168001099.7810872.14547392.458.6012.051251220070511.1D000000S0D000000S0D000000S0D000000S0D711030SN0D703161',
```
Comment (floryst) [inline src/config.ts:152]: This volumeKey should be `1.3.6.1.4.1.5962.1.1.0.0.0.1196533885.18148.0.21.31.200000e0051251220030505.1D000000S0D001152SN0D001332SN0D001144S1D000000S0D006144`.

File: src/config.ts [hunk 2]
```diff
@@ -123,32 +125,50 @@ export const LABELMAP_PALETTE = {
...
+  },
+  {
+    name: 'MRI Cardiac 3D and Cine',
+    filename: 'MRI-Cardiac-3D_and_Cine.zip',
+    description: 'MRI scan with two series: 3D axial non-gated and 2 chamber cine. (4 MB)',
+    url: 'https://data.kitware.com/api/v1/item/6350b28f11dab8142820949d/download',
+    image: MRICardiacThumbnail,
+    volumeKey:
```
Comment (floryst) [inline src/config.ts:164]: Hm, this is two volumes. We can use `1.3.6.1.4.1.5962.99.1.4055780784.1832156800.1666208157104.7.0.3825625620190430.1D000000SN0D000000S0D000000SN0D000000S1D000000S0D000000` as the volume key for now, but detection of whether the sample is loaded will only detect this volume.

File: src/config.ts [hunk 3]
```diff
@@ -123,32 +125,50 @@ export const LABELMAP_PALETTE = {
...
+      '1.2.276.0.50.192168001099.7810872.14547392.458.6012.051251220070511.1D000000S0D000000S0D000000S0D000000S0D711030SN0D703161',
+  },
+  {
+    name: 'MRI PROSTATEx',
+    filename: 'MRI-PROSTATEx-0004.zip',
+    description: 'MRI from the SPIE-AAPM-NCI PROSTATEx challenge. Click application help "(?)" icon for more info. (3 MB)',
+    url: 'https://data.kitware.com/api/v1/item/63527c7311dab8142820a338/download',
+    image: MRIPROSTATExThumbnail,
+    volumeKey:
+      '1.2.276.0.50.192168001099.7810872.14547392.458.6012.051251220070511.1D000000S0D000000S0D000000S0D000000S0D711030SN0D703161',
```
Comment (floryst) [inline src/config.ts:175]: This volumeKey is `1.3.6.1.4.1.14519.5.2.1.7311.5101.206828891270520544417996275680.5tse2d1254.538438420111018.1D000000SN0D000000S0D000000S0D000000S0D970296SN0D241922`.

**Comment by floryst** [comment]
If you could run the new sample dataset images through a jpeg optimizer, that would be great. We are targeting under 5kb per image, with less being better.

**Comment by floryst** [comment]
LGTM! If you can squash your changes (so we don't have a commit containing the large JPGs), we can merge.

**Comment by aylward** [comment]
Can you use the squash and merge button on GitHub?

**Comment by floryst** [comment]
Done!

**Pull request merged**